File: 002-even-fibonacci-numbers/evenFibonacciNumbers.js
let num = process.argv[2];

function fiboEvenSum(n) {
  let i = 1;
  let x = 1;
  let y = 1;
  let sum = 0;
  while (i < n) {
    let z = x + y;
    if (z % 2 === 0) {
      sum += z;
    }
    x = y;
    y = z;
    i++;
  }
  return sum;
}

console.log(fiboEvenSum(num));

File: 002-even-fibonacci-numbers/even_fibonacci_numbers.py
import sys

num = int(sys.argv[1])


def fibo_even_sum(n):
    i = 1
    x = 1
    y = 1
    sum = 0
    while i < n:
        z = x + y
        if z % 2 == 0:
            sum += z
        x = y
        y = z
        i += 1
    return sum


print(fibo_even_sum(num))
